docs/component-adapter/component-inventory.md

```diff
@@ -47,590 +48,589 @@
-| **primaryActionLabel**         | `string`          | Yes      | Text label for the primary action button                                          |
-| **closeActionLabel**           | `string`          | Yes      | Text label for the close/cancel action button                                     |
-| **title**                      | `React.ReactNode` | No       | Optional title content to be displayed at the top of the dialog                   |
-| **children**                   | `React.ReactNode` | No       | Optional children content to be rendered in the dialog body                       |
-| **shouldCloseOnBackdropClick** | `boolean`         | No       | Whether clicking the backdrop should close the dialog                             |
+| Prop | Type | Required | Description |
+|------|------|----------|-------------|
+| **isOpen** | `boolean` | No | Controls whether the dialog is open or closed |
+| **onClose** | `() => void` | No | Callback function called when the dialog should be closed |
+| **onPrimaryActionClick** | `() => void` | No | Callback function called when the primary action button is clicked |
+| **isDestructive** | `boolean` | No | Whether the primary action is destructive (changes button style to error variant) |
+| **isPrimaryActionLoading** | `boolean` | No | Whether the primary action button is in loading state |
+| **primaryActionLabel** | `string` | Yes | Text label for the primary action button |
+| **closeActionLabel** | `string` | Yes | Text label for the close/cancel action button |
+| **title** | `React.ReactNode` | No | Optional title content to be displayed at the top of the dialog |
+| **children** | `React.ReactNode` | No | Optional children content to be rendered in the dialog body |
+| **shouldCloseOnBackdropClick** | `boolean` | No | Whether clicking the backdrop should close the dialog |
 
 ## HeadingProps
 
-| Prop          | Type                                           | Required | Description                                                               |
-| ------------- | ---------------------------------------------- | -------- | ------------------------------------------------------------------------- |
-| **as**        | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6"` | Yes      | The HTML heading element to render (h1-h6)                                |
-| **styledAs**  | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6"` | No       | Optional visual style to apply, independent of the semantic heading level |
-| **textAlign** | `"start" \| "center" \| "end"`                 | No       | Text alignment within the heading                                         |
-| **children**  | `React.ReactNode`                              | No       | Content to be displayed inside the heading                                |
-| **className** | `string`                                       | No       | -                                                                         |
-
-## InputProps
```

Comment:
This is an internal component that isn't actually available for adapters, this correctly removes it